### PR TITLE
Feature/debug argument on debug level

### LINKED ISSUE #335: Command line -debug argument not to raise the level to Trace, but only to Debug

### PR DESCRIPTION
**Pull request by Aprogiena**
-debug now raises to Debug level only, which allows us to use silent Trace logs to file only and not to have too many things going to console with -debug enabled

Fix #335

### CODE REVIEW

File: Stratis.Bitcoin/Configuration/Logging/LoggingConfiguration.cs [hunk 1]
```diff
@@ -40,9 +71,6 @@ static LoggingConfiguration()
 
             // Installs handler to be called when NLog's configuration file is changed on disk.
             LogManager.ConfigurationReloaded += NLogConfigurationReloaded;
-
-            // Load the key to name-space mappings.
-            keyCategories = LoadKeyCategories();
```
Comment (dangershony) [inline Stratis.Bitcoin/Configuration/Logging/LoggingConfiguration.cs]: You didn't like it in the static constructor?

**Comment by dangershony** [comment]
Hi seems there is a conflict can you fix it?

**Comment by Aprogiena** [comment]
Yes, I'll fix tomorrow

File: Stratis.Bitcoin/Configuration/Logging/LoggingConfiguration.cs (hunk 1 above)
Comment (Aprogiena) [inline Stratis.Bitcoin/Configuration/Logging/LoggingConfiguration.cs]: You can see I've deleted quite a few lines after moving it upstairs. So it looks cleaner now.

**Comment by Aprogiena** [comment]
I hate those conflicts :(

**Pull request merged**